lib/generators/boring/rack_cors/install/install_generator.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+module Boring
+  module RackCors
+    class InstallGenerator < Rails::Generators::Base
+      desc 'Adds rack-cors gem to the app'
+      source_root File.expand_path("templates", __dir__)
+
+      class_option :origins, type: :string, default: '*',
+                   desc: 'Specify which origins are allowed to make CORS requests'
```

Comment:
ask user to add multiple origins with space separated value

Reply:
^ should accept an array